tests/toolkit/infile_converters_tests.cc

```diff
@@ -86,11 +86,12 @@ TEST(InfileConverters, JsonPyRoundTrip) {
   cyclus::PyStart();
   string inp = cyclus::toolkit::XmlToJson(MakeInput());
   string p1 = cyclus::toolkit::JsonToPy(inp);
-  string j1 = cyclus::toolkit::PyToJson(p1);
+  string j1 = cyclus::toolkit::PyToJson(p1) + "\n";
```

Comment:
Is this all it takes to get this test working? Why?

Reply:
It took a little more, but not much. There was an inconsistency between two ways of wrapping lines for JSON files.  One way always put a single entry on each line and other way would put a comma separated lists of entries if they fit on a line.  I was able to change the latter by saying lines had to wrap if they were longer than length 1.  There was still a discrepancy of the new line at the end and this was the easiest way to address it for now.